.travis.yml

```diff
@@ -38,10 +42,21 @@ before_script:
   # Disable services enabled by default to avoid port clashing
   - sudo /etc/init.d/mysql stop
   - sudo /etc/init.d/postgresql stop
+  # Get Ansible Vault
+  - sudo pip install ansible
+
 
 # script always runs to completion (set +e). If we have linter issues AND a
 # failing test, we want to see both. Configure golangci-lint with a
 # .golangci.yml file at the top level of your repo.
 script:
+  # Put unencrypted secrets in the expected place
+  - echo $VAULT_PW > /home/travis/gopath/src/github.com/CMSgov/bcda-app/.vault_password
+  - bash /home/travis/gopath/src/github.com/CMSgov/bcda-app/ops/secrets --decrypt
+  - cp -fv /home/travis/gopath/src/github.com/CMSgov/bcda-app/shared_files/encrypted/* /home/travis/gopath/src/github.com/CMSgov/bcda-app/shared_files/decrypted/
```

Comment:
Should this be a move rather than a copy, to avoid leaving unencrypted files in the directory?

Reply:
Changed in the latest commit.